.github/workflows/gomod_checker.yml

```diff
@@ -21,6 +21,7 @@ jobs:
       - uses: actions/setup-go@v5
         with:
           go-version-file: 'go.mod'
+          cache: false
```

Comment:
Even with this change, the CI run of `go mod tidy` _still_ says `downloading github.com/ava-labs/coreth v0.13.6-rc.1.0.20240702201359-ba2ce5367874`, referring yet again to [that non-existent commit](https://github.com/ava-labs/coreth/commit/ba2ce53678740a021f2d5b7aa84e5e08b6fd55f5).